internal/awsservice/cloudwatchlogs.go

```diff
@@ -135,6 +137,30 @@ func IsLogGroupExists(logGroupName string) bool {
 	return len(describeLogGroupOutput.LogGroups) > 0
 }
 
+func GetLogStreams(logGroupName string) []types.LogStream {
+	for i := 0; i < logStreamRetry; i++ {
+		describeLogStreamsOutput, err := CwlClient.DescribeLogStreams(ctx, &cloudwatchlogs.DescribeLogStreamsInput{
+			LogGroupName: aws.String(logGroupName),
+			OrderBy:      types.OrderByLastEventTime,
+			Descending:   aws.Bool(true),
+			Limit:        aws.Int32(10),
+		})
+	
+		if err != nil {
+			log.Printf("failed to get log streams for log group: %v - err: %v", logGroupName, err)
+			continue
+		}
+		
+		if len(describeLogStreamsOutput.LogStreams) > 0 {
+			return describeLogStreamsOutput.LogStreams 
+		}
+
+		time.Sleep(10 * time.Second)
+	}
+	
```

Comment:
nit - it would be nice to add a log here that we exhausted retries so we returned nothing